pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -58,6 +58,15 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
 - data[:valid_grades] = DashboardSection::valid_grades
 - data[:disabled_scripts] = DashboardSection::progress_disabled_scripts(request.user_id)
 
+- gdpr_data = {}
```

Comment:
We're getting closer to `:ruby` looking good :)

Reply:
Or extracting a `gdpr_data` view helper in a ruby file, instead of having this logic in HAML.